electron/main.js

```diff
@@ -243,6 +247,14 @@ const createMainWindow = async () => {
     showNotification(title, description || undefined);
   });
 
+  // if app (ie. mainWindow) is loaded, destroy splash window.
+  ipcMain.on('is-app-loaded', (_event, isLoaded) => {
+    if (isLoaded && splashWindow) {
+      splashWindow.destroy();
+      splashWindow = null;
```

Comment:
is it necessary to set it to null? just wondering, if the window is destroyed (i.e., closed), no events will work anyway. but could setting it to null also cause unexpected errors?